src/clients/SpokePoolClient.ts

```diff
@@ -628,7 +682,7 @@ export class SpokePoolClient extends BaseAbstractClient {
     // new deposits that were found in the searchConfig (new from the previous run). This is important as this operation
     // is heavy as there is a fair bit of block number lookups that need to happen. Note this call REQUIRES that the
     // hubPoolClient is updated on the first before this call as this needed the the L1 token mapping to each L2 token.
-    if (eventsToQuery.includes("FundsDeposited")) {
+    if (eventsToQuery.includes("FundsDeposited") || eventsToQuery.includes("V3FundsDeposited")) {
```

Comment:
Do we have to add `V3FundsDeposited` to `queryableEventNames`?

Reply:
I investigated this and it's not really achievable or useful until we bump contracts-v2. We can survive without it when using the MockSpokePoolClient because it overrides the SpokePoolClient._update() method, which is where we use these filters.

Reply:
Yeah, we need to bump contracts-v2 here before we can use it here.